src/NBS.Appointments.Service.Unit.Tests/ControllerTests/SlotControllerTests.cs

```diff
@@ -0,0 +1,123 @@
+﻿using FluentAssertions;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using NBS.Appointments.Service.Controllers;
+using NBS.Appointments.Service.Core.Interfaces.Services;
+using NBS.Appointments.Service.Models;
+using NBS.Appointments.Service.Validators;
+using System.Net;
+using Xunit;
+
+namespace NBS.Appointments.Service.Unit.Tests.ControllerTests
+{
+    public class SlotControllerTests
+    {
+        private Mock<IQflowService> mockQflowService = new();
+        private SlotAvailabilityRequestValidator _validator = new();
+
+        private SlotController _sut;
+
+        public SlotControllerTests()
+        {
+            _sut = new SlotController(
+                mockQflowService.Object,
+                _validator);
+        }
+
+        [Theory]
+        [InlineData("")]
+        [InlineData(" ")]
+        [InlineData(null)]
+        public async Task AvailableSlots_ShouldFailValidation_WhenSiteIdentifierNotProvided(string siteIdentifier)
+        {
+            var request = new SlotAvailabilityRequest
+            {
+                SiteIdentifier = siteIdentifier,
+                AppointmentType = "COVID18TO65",
+                Date = DateTime.Today.AddDays(1)
+            };
+
+            var result = await _sut.AvailableSlotsAsync(request) as BadRequestObjectResult;
```

Comment:
This feels like a repeat of the api tests. I would be tempted to remove them from here